app/views/membership_applications/_search_form.html.haml

```diff
@@ -1,6 +1,17 @@
 = search_form_for @search_params, method: :get, url: membership_applications_path,
                                   class: 'form-horizontal' do |f|
   .form-group
+    .col-sm-2
+      = f.label :user_membership_number_in,
+                "#{t('membership_applications.index.membership_number')}",
+                class: 'control-label',
+                style: 'text-align: left; font-size: 12px;'
```

Comment:
I think this should be a CSS style.  But this could be implemented in a future PR; don't want to hold up this PR because of it.